src/components/search/tests/CurrentRefinements.test.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import { CurrentRefinementsBase } from '../CurrentRefinements';
+
+import { SUBJECTS, AVAILABLILITY, FACET_ATTRIBUTES } from '../data/constants';
+import { renderWithRouter } from '../../../utils/tests';
+
+describe('<CurrentRefinements />', () => {
+  const items = [{
+    attribute: FACET_ATTRIBUTES.SUBJECTS,
+    items: [{ label: SUBJECTS.COMPUTER_SCIENCE }, { label: SUBJECTS.COMMUNICATION }],
+  }, {
+    attribute: FACET_ATTRIBUTES.AVAILABLILITY,
+    items: [{ label: AVAILABLILITY.AVAILABLE_NOW }, { label: AVAILABLILITY.UPCOMING }],
+  }];
+
+  test('renders refinements and supports viewing all active refinements', () => {
+    renderWithRouter(<CurrentRefinementsBase items={items} />);
+
+    // assert first 3 active refinements are visible
+    expect(screen.queryByText(SUBJECTS.COMPUTER_SCIENCE)).toBeInTheDocument();
+    expect(screen.queryByText(SUBJECTS.COMMUNICATION)).toBeInTheDocument();
+    expect(screen.queryByText(AVAILABLILITY.AVAILABLE_NOW)).toBeInTheDocument();
+
+    // assert additional refinement does not show, and a "+1" button shows
+    expect(screen.queryByText(AVAILABLILITY.UPCOMING)).not.toBeInTheDocument();
+    expect(screen.queryByText('+1', { exact: false })).toBeInTheDocument();
```

Comment:
What does `exact: false` do here?

Reply:
@raq929 From the react-testing-library docs: https://testing-library.com/docs/react-testing-library/cheatsheet (at the bottom)